### ISSUE TITLE
Integrate an API

### ISSUE THREAD

**Issue by vipul0425** [vipul0425/Calorie-Tracker#4]
If you want to make a good level contribution you can integrate a food API into this project (eg. Nutritionix) which will tell automatically fill the calories section when the user adds a particular meal.

**Comment by lakshya8066** [comment]
Hey @vipul0425 I would like to work on this. Could you assign me?

**Comment by vipul0425** [comment]
Yes, you can but please make sure to make open PR on the contributions branch.

**Comment by lakshya8066** [comment]
Sure @vipul0425 I will keep that in mind
